### PR TITLE
fix: return correct url if entity name is empty

### PR DESCRIPTION
**Pull request by pleshakovalexander**
Can be useful in case calling method without entity

### CODE REVIEW

File: src/angularODataConfiguration.ts [hunk 1]
```diff
@@ -61,7 +61,10 @@ export class ODataConfiguration {
     }
 
     public getEntitiesUri(typeName: string): string {
-        return `${this.baseUrl}/${this.sanitizeTypeName(typeName)}`;
+        if(typeName){
```
Comment (StefH) [inline src/angularODataConfiguration.ts]: Please do a format file.

File: src/angularODataConfiguration.ts [hunk 2]
```diff
@@ -61,7 +61,10 @@ export class ODataConfiguration {
     }
 
     public getEntitiesUri(typeName: string): string {
```
Comment (StefH) [inline src/angularODataConfiguration.ts:63]: And update / add a unit-test

**Review by StefH — CHANGES_REQUESTED** [review]
see comments...

File: src/angularODataConfiguration.ts (hunk 1 above)
Comment (pleshakovalexander) [reply src/angularODataConfiguration.ts]: @StefH done

File: src/angularODataConfiguration.ts (hunk 2 above)
Comment (pleshakovalexander) [reply src/angularODataConfiguration.ts:63]: @StefH done

**Review by StefH — APPROVED** [review]

**Pull request merged**